Formula/lib/libvirt-python.rb

```diff
@@ -36,14 +37,16 @@ def install
   end
 
   test do
-    system "python3.12", "-c",
-           # language=Python
-           <<~EOS
-             import libvirt
+    pythons.each do |python|
```

Comment:
why do you need pythons in here?

Reply:
I think, it's a good idea to compute the list of Pythons instead of coding the same versions in two or more places.

Reply:
but there is just one python in the dependency list, what is the hassle of hardcoding it?

Reply:
oh, i see, we still keep pythons as before. hmm

make sense then.